scenes/game/game.gd

```diff
@@ -2374,7 +2384,18 @@ func _update_buttons():
 		var card_name = ""
 		if 'card_name' in choice:
 			card_name = choice['card_name']
-		button_choices.append({ "text": CardDefinitions.get_effect_text(choice, false, true, false, card_name), "action": func(): _on_choice_pressed(i) })
+		var card_text = CardDefinitions.get_effect_text(choice, false, true, false, card_name)
+		if len(card_text) > ChoiceTextLengthSoftCap:
+			var break_idx = ChoiceTextLengthSoftCap-1
+			while break_idx < len(card_text)-1 and card_text[break_idx] != " ":
+				break_idx += 1
+				if break_idx >= ChoiceTextLengthHardCap:
+					break
+			if card_text[break_idx] == " ":
+				card_text = card_text.substr(0, break_idx) + "\n" + card_text.substr(break_idx+1)
+			else:
+				card_text = card_text.substr(0, break_idx) + "-\n" + card_text.substr(break_idx)
```

Comment:
Does this still make sense if the last word of the text spanned the soft cap limit?
In that case, break_idx should be = len(card_text)-1, which is the last character.